### PR TITLE
[PWGMM] Added histograms to analyse-mft-tracks

### PR DESCRIPTION
**Pull request by sarahherrmann**
I added a lot of histograms into my histogramRegistry and filled them in processRec

### CODE REVIEW

File: PWGMM/Tasks/analyse-mft-tracks.cxx [hunk 1]
```diff
@@ -27,33 +27,99 @@ struct analyseMFTTracks {
...
-      {"TracksPhiEta_in_coll", "; #varphi; #eta; tracks", {HistType::kTH2F, {{600, -M_PI, M_PI}, {35, -4.5, -1.}}}}, //
-      {"TracksEtaZvtx", "; #eta; Z_{vtx}; tracks", {HistType::kTH2F, {{35, -4.5, -1.}, {201, -20.1, 20.1}}}},        //
-      {"NtrkZvtx", "; N_{trk}; Z_{vtx}; events", {HistType::kTH2F, {{301, -0.5, 300.5}, {201, -20.1, 20.1}}}},       //
-      {"NtrkEta", "#eta; N_{trk}; events", {HistType::kTH1F, {{35, -4.5, -1.}}}},                                    //
-      {"TracksPhiEtaGen", "; #varphi; #eta; tracks", {HistType::kTH2F, {{600, 0, 2 * M_PI}, {35, -4.5, -1.}}}},      //
-      {"TracksEtaZvtxGen", "; #eta; Z_{vtx}; tracks", {HistType::kTH2F, {{35, -4.5, -1.}, {201, -20.1, 20.1}}}},     //
+      {"TracksPhiEta_in_coll", "; #varphi; #eta; ", {HistType::kTH2F, {{600, -M_PI, M_PI}, {35, -4.5, -1.}}}}, //
+      {"TracksPhiEta", "; #varphi; #eta; ", {HistType::kTH2F, {{600, -M_PI, M_PI}, {35, -4.5, -1.}}}},         //
+
+      {"TracksPhiEta_5", "#eta_{kin} versus #varphi_{kin} of MFT tracks with ncls #\geq 5; #varphi; #eta; ", {HistType::kTH2F, {{600, -M_PI, M_PI}, {35, -4.5, -1.}}}}, //
```
Comment (aalkin) [inline PWGMM/Tasks/analyse-mft-tracks.cxx:33]: Why not a TH3?

File: PWGMM/Tasks/analyse-mft-tracks.cxx [hunk 2]
```diff
@@ -27,33 +27,99 @@ struct analyseMFTTracks {
...
   //end of processRec
   PROCESS_SWITCH(analyseMFTTracks, processRec, "Process rec level", true);
 
+  void processTracks(o2::aod::MFTTracks const& tracks)
+  {
+
+    for (auto& track : tracks) {
+      registry.fill(HIST("TracksPhiEta"), track.phi(), track.eta());
+
+      if (track.nClusters() >= 5) {
```
Comment (aalkin) [inline PWGMM/Tasks/analyse-mft-tracks.cxx:103]: Please take a look at `histogramRegistry.cxx` tutorial, lines 111-116, there is a nicer way to fill these histograms.

File: PWGMM/Tasks/analyse-mft-tracks.cxx (hunk 1 above)
Comment (sarahherrmann) [reply PWGMM/Tasks/analyse-mft-tracks.cxx:33]: I personnally think a TH2 is easier to read (for me), can I keep it this way ? :)

File: PWGMM/Tasks/analyse-mft-tracks.cxx (hunk 2 above)
Comment (sarahherrmann) [reply PWGMM/Tasks/analyse-mft-tracks.cxx:103]: Ok will do

**Comment by aalkin** [comment]
Obsolete due to #500

**Pull request closed without merging**